frontend/src/main/web/src/components/Base/readme.md

```diff
@@ -0,0 +1,29 @@
+The base component for most other components. It allows other components to
+inherit it's props, which are useful for styling with atomic classes.
+
+It should never be used directly, only through another component.
+
+## Example of Base in <Icon />
+
+`<Icon />` uses <Base /> as it's core component.
```

Comment:
This looks like it is missing information about how the default styles would be specified in the `Icon` component.